go/v1beta1/storage/elasticsearch.go

```diff
@@ -121,7 +198,37 @@ func (es *ElasticsearchStorage) ListProjects(ctx context.Context, filter string,
 }
 
 // DeleteProject deletes the project with the given pID from the store
-func (es *ElasticsearchStorage) DeleteProject(ctx context.Context, pID string) error {
+func (es *ElasticsearchStorage) DeleteProject(ctx context.Context, projectID string) error {
+	projectName := fmt.Sprintf("projects/%s", projectID)
```

Comment:
delete should also remove note and occurrence indices

Reply:
Fun fact! Not only did grafeas delete the project and return a 500, it also used to leave rogue notes and occurrences too!